symfony-cmf/seo-bundle/2.0/config/packages/cmf_seo_bundle.yaml

```diff
@@ -0,0 +1,7 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/seo/configuration.html
+cmf_seo:
+    persistence:
+        phpcr:
+            enabled: true
+    title: %%content_title%% | Symfony2 CMF
+    description: The Symfony2 Content Management Framework. %%content_description%%
```

Comment:
Both title and description seem pretty CMF application specific? (and the values should be between single quotes)

Reply:
yes, it is. but what else as dummy data?

Reply:
that are values every user would configure.